src/convertToObject.js

```diff
@@ -5,8 +5,23 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(styles) {
+  const styleObject = {};
+
+  // Поділ  рядків крапками з комою
+  const declarations = styles.split(';');
+
+  declarations.forEach((declaration) => {
+    // Розділ кожну декларацію двокрапкою
+    const [property, ...values] = declaration.split(':');
+
+    // Перевірте на існ  власт, так і значн, і видал зайві пробіли
+    if (property && values.length > 0) {
+      styleObject[property.trim()] = values.join(':').trim();
+    }
+  });
+
+  return styleObject;
```

Comment:
The function `convertToObject` is well-structured and follows the task requirements effectively. However, it would be beneficial to consider edge cases where multiple colons might exist within a property's value or where multiple semicolons occur without any styles between them. While not required, handling these cases would make the function more robust.